src/FTPClient.jl

```diff
@@ -33,6 +35,16 @@ export RequestOptions,
     binary_mode,
     close
 
+const C_WRITE_FILE_CB = Ref{Ptr{Void}}(C_NULL)
```

Comment:
Judging from the code below, apparently there's a `Ptr{Cvoid}` now so we should use that. TIL!